src/styles/box/card.scss

```diff
@@ -0,0 +1,168 @@
+.catalog {
+  display: grid;
+  grid-template-rows: 408px;
+  grid-template-columns: 200px;
+  padding: 50px 40px;
+  gap: 46px 48px;
+
+  @media screen and (min-width: 488px) {
+    grid-template-columns: repeat(2, 1fr);
+  }
+
+  @media screen and (min-width: 768px) {
+    grid-template-columns: repeat(3, 1fr);
+  }
+
+  @media screen and (min-width: 1024px) {
+    grid-template-columns: repeat(4, 1fr);
+  }
+
+  @media (min-width: 1200px) {
+    grid-template-columns: repeat(4, 1fr);
+    padding-left: 128px;
+    padding-right: 128px;
+  }
+}
+
+.product-card {
+  box-sizing: border-box;
+  width: $card-width;
+  height: 412px;
+  display: flex;
+  align-items: center;
+  flex-flow: column wrap;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  background-color: #fff;
+  text-align: center;
+  margin: 0;
+  padding: 32px 0 7px;
+
+  &__bg {
+    width: 160px;
+    height: 134px;
+    margin: 0 auto;
+    margin-bottom: 40px;
+    padding: 0 19px;
+  }
+
+  &__title {
+    width: 166px;
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 18px;
+    text-align: left;
+    text-underline-position: from-font;
+    text-decoration-skip-ink: none;
+    color: #060b35;
+    margin: 0;
+  }
+
+  &__title:hover {
+    color: #34568b;
+  }
+
+  &__code {
+    font-size: 10px;
+    font-weight: 400;
+    line-height: 14px;
+    text-align: left;
+    text-underline-position: from-font;
+    text-decoration-skip-ink: none;
+    color: #616070;
+    margin-top: 4px;
+    margin-bottom: 0;
+    margin-right: 68px;
+  }
+
+  &__rating {
+    width: 169px;
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    margin: 16px 0 2px 3px;
+    font-family: Roboto, sans-serif;
+    font-size: 10px;
+    font-weight: 400;
+    line-height: 14px;
+    text-align: right;
+    text-underline-position: from-font;
+    text-decoration-skip-ink: none;
+    color: #060b35;
+  }
+
+  &__reviews {
+    width: 52px;
+    display: flex;
+    box-sizing: border-box;
+    text-align: right;
+    padding-top: 1px;
+    padding-bottom: 0;
+    padding-left: 2px;
+    margin-left: 16px;
+  }
+
+  &__price {
+    width: 166px;
+    display: flex;
+    justify-content: space-between;
+    margin-top: 21px;
+    margin-bottom: 15px;
+
+    &-title {
+      font-size: 12px;
+      font-weight: 400;
+      line-height: 18px;
+      text-align: left;
+      text-underline-position: from-font;
+      text-decoration-skip-ink: none;
+      color: #616070;
+      margin-bottom: 0;
+      padding-top: 2px;
+    }
+
+    &-sum {
+      box-sizing: border-box;
+      font-size: 16px;
+      font-weight: 700;
+      line-height: 18px;
+      text-align: right;
```

Comment:
The `text-align: right;` property is redundant here because it is immediately overridden by `text-align: end;`. Consider removing one of these properties to avoid redundancy.